packages/ra-ui-materialui/src/input/DateTimeInput.tsx

```diff
@@ -33,7 +33,7 @@ const dateTimeRegex = /^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}$/;
  *
  * @param {Date | String} value date string or object
  */
-const format = (value: string | Date) => {
+const dateTimeFormatter = (value: string | Date) => {
```

Comment:
I think it would be clearer to not naming like a class. Something like `formatDateTime` ?

Reply:
Done!